bluepyopt/ephys/morphologies.py

```diff
@@ -24,6 +24,8 @@
 import os
 import platform
 import logging
+import numpy
+import arbor
```

Comment:
Could we make this conditional, or another option is to put the arbor morph in a separate file should that only needs to be importanted when necessary

Reply:
I'm importing arbor now locally in the functions that require it and raise an appropriate error if it's not installed.